pkgs/by-name/re/renode/common.nix

```diff
@@ -0,0 +1,118 @@
+{
```

Comment:
Is it better to include that as a function in the original recipe and pass it through externally, so the other recipes can use it instead of adding it directly as a file?